de.js

```diff
@@ -0,0 +1,334 @@
+    you_havent_created_a_cloud_backup: `Sie haben kein Cloud-Backup erstellt`,
+    manage: `Verwalten`,
+    backup: `Sicherung`,
+    view_seed_phrase: `Seed-Phrase anzeigen`,
+    pin: `STIFT`,
+    back_up_to: `Sichern Sie`,
+    back_up_your_wallet: `Sichern Sie Ihr Wallet`,
+    back_up_your_wallet_description: `Speichern Sie Ihr Wallet in der Cloud und schützen Sie es mit einem Passwort. Sie können Ihre Seed-Phrase auch aufschreiben oder in einen Passwort-Manager kopieren.`,
+    pin_description: `Wenn Sie Ihre PIN vergessen, müssen Sie Ihr Wallet aus einer Sicherungskopie wiederherstellen.`,
+    change_pin: `PIN ändern`,
+    your_pin_is_required: `Für Zahlungen und vertrauliche Aktionen ist Ihre PIN erforderlich.`,
+    use_fingerprint: `Fingerabdruck/Gesichtserkennung verwenden`,
+    ask_every_time_i_open_the_app: `Jedes Mal fragen, wenn ich die App öffne`,
+    this_will_remove_all_wallet: `Dadurch werden alle mit der Brieftasche in Zusammenhang stehenden Daten von Ihrem Gerät entfernt.`,
+    delete_wallet: `Wallet löschen`,
+    sign_in_to_the_dogeverse: `Melden Sie sich beim Dogeverse an`,
+    dogeverse_intro_description: `Fordern Sie Ihren DogeTag an, finden Sie Freunde, erhalten Sie Zahlungen mit Ihrem Link und interagieren Sie mit Inhalten im Dogeverse.`,
+    sign_in: `anmelden`,
+    sign_in_description: `Ihre Telefonnummer wird nur zur Anmeldung verwendet und nicht veröffentlicht.`,
+    create_account_description: `Treten Sie dem Dogeverse bei und verbinden Sie sich mit einem riesigen Netzwerk von Shibes.`,
+    send_verification: `Bestätigung senden`,
+    verify_your_phone_number: `Bestätige deine Telefonnummer`,
+    enter_the_code_sent_to: `Geben Sie den Code ein, der gesendet wurde an`,
+    resend_code: `Code erneut senden`,
+    wrong_number: `Falsche Nummer`,
+    confirm_code: `Code bestätigen`,
+    youve_entered_the_verse: `Du bist in den Vers eingetreten`,
+    youre_now_ready_to_explore_the_dogeverse: `Jetzt sind Sie bereit, das Dogeverse zu erkunden!`,
+    lets_go: `Lass uns gehen!`,
+    edit_profile: `Profil bearbeiten`,
+    social_connect: `Soziale Verbindung`,
+    profile_privacy: `Profil Datenschutz`,
+    profile_is_public: `Profil ist öffentlich`,
+    show_wallet_address_and_tip_button: `Wallet-Adresse und Trinkgeld-Button anzeigen`,
+    preferences: `Einstellungen`,
+    confirm_x_tips: `Bestätigen Sie 𝕏 Tipps`,
+    tips_received: `Erhaltene Trinkgelder`,
+    new_followers: `Neue Follower`,
+    new_memes: `Neue Memes`,
+    about_mydoge: `Über MyDoge`,
+    terms_of_use: `Nutzungsbedingungen`,
+    privacy_policy: `Datenschutzrichtlinie`,
+    share_mydoge: `MyDoge teilen`,
+    visit_our_website: `Besuche unsere Webseite`,
+    settings_and_support: `Einstellungen und Support`,
+    connect_twitch: `Twitch verbinden`,
+    connect_x: `Verbinden 𝕏`,
+    create_a_post: `Einen Beitrag erstellen`,
+    following: `Gefolgt`,
+    followers: `Anhänger`,
```

Comment:
okay, better as on X since 'Follower' is well-known in German too... same in singular and plural -> 
`    followers: 'Follower',`